usr/share/rear/prep/NETFS/default/05_check_NETFS_requirements.sh

```diff
@@ -78,16 +81,31 @@ bzip2
 xz
 )
 
-if [[ "$scheme" = "sshfs" ]] ; then
-    # see http://sourceforge.net/apps/mediawiki/fuse/index.php?title=SshfsFaq
-    REQUIRED_PROGS=( "${REQUIRED_PROGS[@]}" sshfs )
+# include required stuff for sshfs or ftpfs (via CurlFtpFS)
+if [[ "sshfs" = "$scheme" || "ftpfs" = "$scheme" ]] ; then
+    # both sshfs and ftpfs (via CurlFtpFS) are based on FUSE
     PROGS=( "${PROGS[@]}" fusermount mount.fuse )
     MODULES=( "${MODULES[@]}" fuse )
     MODULES_LOAD=( "${MODULES_LOAD[@]}" fuse )
-    # as we're using SSH behind the scenes we need our keys/config file saved
-    COPY_AS_IS=( "${COPY_AS_IS[@]}" $HOME/.ssh /etc/fuse.conf )
+    COPY_AS_IS=( "${COPY_AS_IS[@]}" /etc/fuse.conf )
+    # include what is specific for sshfs
+    if [[ "sshfs" = "$scheme" ]] ; then
+        # see http://sourceforge.net/apps/mediawiki/fuse/index.php?title=SshfsFaq
+        REQUIRED_PROGS=( "${REQUIRED_PROGS[@]}" sshfs )
+        # as we're using SSH behind the scenes we need our keys/config file saved
+        COPY_AS_IS=( "${COPY_AS_IS[@]}" $HOME/.ssh )
+    fi
+    # include what is specific for ftpfs
+    if [[ "ftpfs" = "$scheme" ]] ; then
+        # see http://curlftpfs.sourceforge.net/
+        # and https://github.com/rear/rear/issues/845
+        REQUIRED_PROGS=( "${REQUIRED_PROGS[@]}" curlftpfs )
+    fi
 fi
 
 # include required modules, like nfs cifs ...
+# FIXME: the code below includes modules for all non-empty schemes
```

Comment:
@jsmeix this must been seen as a catch all rule (as you never know what you could miss)